scripts/TestHarness/testers/RavenErrors.py

```diff
@@ -18,6 +18,9 @@
 import subprocess
 import platform
 
+fileDir = os.path.dirname(os.path.realpath(__file__))
+raven = os.path.abspath(os.path.join(fileDir,'..','..','..','framework','Driver.py'))
+
```

Comment:
I abstracted the path determination so that `RavenError` can be run from ANY of the tests directories, not just one.